src/js/shot-generator/SelectionManager.js

```diff
@@ -52,14 +52,58 @@ function getObjectsFromCameraView (objects) {
         results = results.concat(o.bonesHelper.hit_meshes)
       }
     }
+    if(o.userData.type === 'controlTarget')
+    {
+      if(o.children.length !== 0)
+      {
+        let gizmo = o.children[0];
+        let gizmoChildren = gizmo.children;
+        for(let i = 0; i < gizmoChildren.length; i++)
+        {
+          if(gizmoChildren[i].name === "Helper")
+          {
+              let children = gizmoChildren[i].children;
+              for (let i = 0; i < children.length; i++)
+              {
+                results.push(children[i]);
+              }
+          }
+        }
+      }
+    }
+
+    if(o.userData.type === 'boneControl')
+    {
+      if(o.children.length !== 0)
+      {
+        let gizmo = o.children[0];
+        let gizmoChildren = gizmo.children;
+        for(let i = 0; i < gizmoChildren.length; i++)
+        {
+          if(gizmoChildren[i].name === "Helper")
+          {
+              let children = gizmoChildren[i].children;
+              for (let i = 0; i < children.length; i++)
+              {
+                children[i].userData.type = "boneControl";
+                results.push(children[i]);
+              }
+          }
+        }
+      }
+    }
 
+    if(o.userData.type === 'controlPoint')
+    {
+      results.push(o);
+    }
     // don't allow selection of: camera, volume
   }
 
   return results
 }
 
-const getIntersectionTarget = intersect => {
+const getIntersectionTarget = (intersect, exceptGizmo) => {
```

Comment:
I don't think the `exceptGizmo` logic belongs in `getIntersectionTarget`. Should probably exclude gizmo/boneControl before getIntersectionTarget is even called.